### PR TITLE
[DPE-5244] Split PITR backup test in AWS and GCP

### PR DESCRIPTION
**Pull request by lucasgameiroborges**
## Issue

PITR backup tests should run in parallel on both GCP and AWS clouds.

## Solution

- Refactor (similar to non-PITR backup tests) to a helper function.
- 2 tests in different groups.

### CODE REVIEW

File: tests/integration/helpers.py [hunk 1]
```diff
@@ -1029,3 +1030,197 @@ async def backup_operations(
...
+async def pitr_backup_operations(
+    ops_test: OpsTest,
+    s3_integrator_app_name: str,
+    tls_certificates_app_name: str,
+    tls_config,
+    tls_channel,
+    credentials,
+    cloud,
+    config,
+) -> None:
```
Comment (dragomirp) [inline tests/integration/helpers.py:1044]: IMHO this should stay in the PITR test as a hanging functions, since it's not really reusable by anything else.

**Review by dragomirp — APPROVED** [review]

File: tests/integration/helpers.py (hunk 1 above)
Comment (marceloneppel) [reply tests/integration/helpers.py:1044]: I agree. We can move this now to there or later—no need to block the PR.

**Review by marceloneppel — APPROVED** [review]
LGTM!

File: tests/integration/helpers.py (hunk 1 above)
Comment (lucasgameiroborges) [reply tests/integration/helpers.py]: Should be better now, lighter diff too :)

**Review by taurus-forever — APPROVED** [review]

**Pull request merged**